### ISSUE TITLE
`no-extraneous-args` better report methods with alternatives

### ISSUE THREAD

**Issue by jfmengels** [jfmengels/eslint-plugin-lodash-fp#16]
I figure that for a lot of people starting with Lodash/fp will try stuff like `_.get(defaultValue, path, object)` with arguments in some order, only to find out that it has its arity fixed at 2.

With `no-extraneous-args`, you get a warning that one of the arguments will be ignored. What would be nice, is to take it one step further and to check if the method has an alias that doesn't ignore that argument.

Example:
```js
`_.get(defaultValue, path, object);
// => Too many arguments. Are you looking for `_.getOr`?
```

The message should be improved, but that's the gist of it. I think it's fine to only display if the number of args matches exactly the number of arguments of the alias (e.g. `_.get(a, b, c, d)` would get the regular message).
